components/deny_public_fields/lib.rs

```diff
@@ -17,7 +17,11 @@ fn expand_string(input: &str) -> String {
     let style = synstructure::BindStyle::Ref.into();
     synstructure::each_field(&type_, &style, |binding| {
         if binding.field.vis != syn::Visibility::Inherited {
-            panic!("Field {} should not be public", binding.ident);
+            panic!("Field `{}` should not be public",
+                   match binding.field.ident.clone() {
```

Comment:
We can write this as `panic!("Field `{}` should not be public", binding.field.ident.as_ref().unwrap_or(&binding.ident));` which is cleaner and avoids clone.